backfill.sql

```diff
@@ -181,8 +189,17 @@ BEGIN
             $$;
     END IF;
 
-    IF on_conflict_do_nothing THEN
+    IF on_conflict_action = 'DoNothing' THEN
         on_conflict_clause = 'ON CONFLICT DO NOTHING';
+    ELSEIF on_conflict_action = 'Update' THEN
+        on_conflict_clause = 'ON CONFLICT DO UPDATE SET ';
+        FOR on_conflict_index IN 1 .. ARRAY_LENGTH(update_action_columns, 1) LOOP
+            on_conflict_column = update_action_columns[on_conflict_index];
+            on_conflict_clause = on_conflict_clause || FORMAT('%2$s.%1$I = to_insert.%1$I', on_conflict_column, dest);
```

Comment:
I think the format clause will be simpler if you do something like `FORMAT('%1$I = EXCLUDED.%1$I', on_conflict_column)`